CONTRIBUTING.md

```diff
@@ -34,6 +34,19 @@ We'll try to be as responsive as possible in reviewing and accepting pull reques
 - If `index.yaml` gets out of date on the repository, see
   [`./scripts/`](./scripts) for a workflow to fix
 
+## Testing
+
+Running the [helm chart unit tests](https://github.com/helm-unittest/helm-unittest):
+
+```
+# install the unittest plugin
+helm plugin install https://github.com/helm-unittest/helm-unittest.git
+
+# run the rstudio-connect chart unittests
```

Comment:
Maybe make this a little more generic so people know it's not just for `rstudio-connect`?

Reply:
Added a `just test` target to our justfile